distribution/docs/src/main/resources/_contents/_developing/developing-intro-contents.adoc

```diff
@@ -0,0 +1,816 @@
+    <bundle>mvn:org.codice.thirdparty/picocontainer/1.2_1</bundle>
+    <bundle>mvn:org.codice.thirdparty/vecmath/1.3.2_1</bundle> <!-- for GeoTools -->
+    <bundle>mvn:org.codice.thirdparty/geotools-suite/${org.geotools.bundle.version}</bundle>
+    <bundle>mvn:org.codice.thirdparty/jts/${jts.bundle.version}</bundle>
+    <bundle>mvn:${ddf-branding-lowercase}.catalog.core/catalog-core-federationstrategy/${ddf.version}</bundle>
+    <bundle>mvn:org.codice.thirdparty/lucene-core/3.0.2_1</bundle>
+    <bundle>mvn:${ddf-branding-lowercase}.catalog.core/${ddf-branding-lowercase}-pubsub/${ddf.version}</bundle>
+    <bundle>mvn:${ddf-branding-lowercase}.catalog.core/catalog-core-eventcommands/${ddf.version}</bundle>
+    <bundle>mvn:${ddf-branding-lowercase}.catalog.core/${ddf-branding-lowercase}-pubsub-tracker/${ddf.version}</bundle>
+    <bundle>mvn:${ddf-branding-lowercase}.catalog.core/catalog-core-urlresourcereader/${ddf.version}</bundle>
+    <bundle>mvn:${ddf-branding-lowercase}.catalog.core/filter-proxy/${ddf.version}</bundle>
+    <bundle>mvn:${ddf-branding-lowercase}.catalog.core/catalog-core-commands/${ddf.version}</bundle>
+    <bundle>mvn:${ddf-branding-lowercase}.catalog.core/catalog-core-metacardgroomerplugin/${ddf.version}</bundle>
+    <bundle>mvn:${ddf-branding-lowercase}.catalog.core/metacard-type-registry/${ddf.version}</bundle>
+    <bundle>mvn:${ddf-branding-lowercase}.catalog.core/catalog-core-standardframework/${ddf.version}</bundle>
+    <bundle>mvn:${ddf-branding-lowercase}.catalog.core/catalog-core-resourcesizeplugin/${ddf.version}</bundle>
+
+    <configfile finalname="/data/solr/metacard_cache/conf/solrconfig.xml">mvn:ddf.platform.solr/platform-solr-server-standalone/${ddf.version}/xml/solrconfig</configfile>
+    <configfile finalname="/data/solr/metacard_cache/conf/schema.xml">mvn:ddf.platform.solr/platform-solr-server-standalone/${ddf.version}/xml/schema</configfile>
+    <configfile finalname="/data/solr/metacard_cache/conf/protwords.txt">mvn:ddf.platform.solr/platform-solr-server-standalone/${ddf.version}/txt/protwords</configfile>
+    <configfile finalname="/data/solr/metacard_cache/conf/stopwords_en.txt">mvn:ddf.platform.solr/platform-solr-server-standalone/${ddf.version}/txt/stopwords_en</configfile>
+    <configfile finalname="/data/solr/metacard_cache/conf/stopwords.txt">mvn:ddf.platform.solr/platform-solr-server-standalone/${ddf.version}/txt/stopwords</configfile>
+    <configfile finalname="/data/solr/metacard_cache/conf/synonyms.txt">mvn:ddf.platform.solr/platform-solr-server-standalone/${ddf.version}/txt/synonyms</configfile>
+</feature>
+----
+
+==== Making Sure a Features File Will Display Properly in the Installer
+
+In order to ensure that the installer can correctly interpret and display application details, there are several guidelines that should be followed when creating the features file for the application.
+
+* Be sure that only one feature in the `features.xml` has the `auto-install` tag.
```

Comment:
This isn't true anymore. @kcwire, could you provide a quick explanation to Rick on how the `auto` flag is interpreted by our app installer now?